src/main/webapp/app/markdown-editor/commands/orderedListCommand.ts

```diff
@@ -31,24 +31,26 @@ export class OrderedListCommand extends Command {
     }
 
     /**
-     * @function execute
-     * @desc 1. Check if the selected text includes (.)
-     *       2. If included reduce the selected text by this element and replace the selected text by textToAdd
-     *       3. If not included place the position before the selected text and add them to the editor
+     * @function replaceText
+     * @desc 1. Check if the selected text includes (.) because the ordered counting includes always a number followed by a dot
+     *       2. If included, reduce the selected text by 3 (number, dot, whitespace) and replace the selected text by textToAdd
+     *       3. If not included, place the position {number} before the selected text {string} and add them to the editor
      *       4. An ordered list in markdown language appears
-     * @param {string} with the position {number} it has in the array
+     * @param extracted textLine {string} with the position {number} it has in the overall selectedText{array}
      */
     replaceText(element: string, position: number): void {
         if (element.includes('.')) {
             const textToAdd = element.slice(3);
             const text = `${textToAdd}\n`;
             this.insertText(text);
+            /** check if it is an empty string an start the basic command of an ordering list with number one*/
         } else if (element === '') {
             const range = this.getRange();
             element = `1. ${element}`;
```

Comment:
Couldn't the element in the string template be removed here? It should be empty anway.